addons/website_slides/models/slide_slide.py

```diff
@@ -375,16 +375,16 @@ def _compute_embed_counts(self):
         for slide in self:
             slide.embed_count = mapped_data.get(slide.id, 0)
 
-    @api.depends('slide_ids.sequence', 'slide_ids.slide_category', 'slide_ids.is_published', 'slide_ids.is_category')
+    @api.depends('slide_ids.sequence', 'slide_ids.slide_category', 'slide_ids.is_published', 'slide_ids.is_category', 'slide_ids.completion_time', 'slide_ids.active')
```

Comment:
The "completion_time" field should have this method in its definition then.

Reply:
True, corrected that part :smile:

Reply:
Hey :) Does it really need to have this method in the definition, considering that it is actually computed in multiple different places?  (What I mean is that category completion_time is calculated here, google_drive,vimeo and youtube completion_time are calculated in their respective functions, document completion_time is calculated in the separate function, etc?) 

The issue seems to be that if we assign this method to the completion time, then we also need to asign "recursive=True" value to it, as we are calculating the category completion time by summing up the other completion time values, and then we break the cascade deletion functionality on the course :smile:

Reply:
Same issue, separating methods didn't help it :confused: